core/src/main/java/org/radargun/stats/BasicOperationStats.java

```diff
@@ -16,7 +16,8 @@
    private static final double INVERSE_NORMAL_95 = 1.96;
    private static final double INVERSE_NORMAL_50 = 0.67448;
    private long requests;
-   private long responseTimeMax = Long.MIN_VALUE;
+   // in the report it will show -9223372036854775808 if a node doesn't receive any request
```

Comment:
Not sure you need the comment any more...

Reply:
Done